frame/balances/e2e-tests/src/lib.rs

```diff
@@ -0,0 +1,148 @@
+        <T::Runtime as pallet_balances::Config>::Balance: From<u8>,
+        <T::Runtime as frame_system::Config>::AccountId: From<AccountId32> + Encode,
+        <T::Block as BlockT>::Extrinsic: From<Extrinsic<T::Runtime, T::SignedExtras>>,
+        <<T::Runtime as frame_system::Config>::Lookup as sp_runtime::traits::StaticLookup>::Source: From<AccountId32>,
+        AccountId32: std::borrow::Borrow<<T::Runtime as frame_system::Config>::AccountId>
+{
+    type Balances<T> = pallet_balances::Module<T>;
+    let (alice, bob) = (
+        MultiSigner::from(Alice.public()).into_account(),
+        MultiSigner::from(Bob.public()).into_account(),
+    );
+    let (alice_balance, bob_balance) = node.with_state(|| (
+        Balances::<T::Runtime>::free_balance(alice.clone()),
+        Balances::<T::Runtime>::free_balance(bob.clone()),
+    ));
+    let balance = alice_balance / 2u8.into();
+
+    let balances_call = BalancesCall::force_transfer(alice.clone().into(), bob.clone().into(), balance.into());
+    node.submit_extrinsic(
+        SudoCall::sudo(Box::new(balances_call.into())),
+        alice.into()
+    );
+    node.seal_blocks(1);
+
+    // TODO: figure out events
+    // let events = node.events()
+    //     .into_iter()
+    //     .filter(|event| {
+    //         match &event.event {
+    //             Event::pallet_balances(RawEvent::Transfer(_, _, _)) => true,
+    //             _ => false,
+    //         }
+    //     })
+    //     .collect::<Vec<_>>();
+    // assert_eq!(events.len(), 1);
+
+    let new_bob_balance = node.with_state(|| Balances::<T::Runtime>::free_balance(bob.clone()));
+
+    assert_eq!(new_bob_balance, bob_balance + (balance))
+}
+
+/// Tests the Balances::set_balance call
+pub fn set_balance<T>(node: &Node<T>)
+    where
+        T: ChainInfo,
+        T::Runtime: pallet_sudo::Config + pallet_balances::Config,
+        <T::Runtime as frame_system::Config>::Call: From<SudoCall<T::Runtime>> + From<BalancesCall<T::Runtime>> + Encode,
+        <T::Runtime as pallet_sudo::Config>::Call: From<BalancesCall<T::Runtime>>,
+        <T::Runtime as pallet_balances::Config>::Balance: From<u8>,
+        <T::Runtime as frame_system::Config>::AccountId: From<AccountId32> + Encode,
+        <T::Block as BlockT>::Extrinsic: From<Extrinsic<T::Runtime, T::SignedExtras>>,
+        <<T::Runtime as frame_system::Config>::Lookup as sp_runtime::traits::StaticLookup>::Source: From<AccountId32>,
+        AccountId32: std::borrow::Borrow<<T::Runtime as frame_system::Config>::AccountId>,
+{
+    type Balances<T> = pallet_balances::Module<T>;
+    let (alice, bob) = (
+        MultiSigner::from(Alice.public()).into_account(),
+        MultiSigner::from(Bob.public()).into_account(),
+    );
+    let (alice_balance, _bob_balance) = node.with_state(|| (
+        Balances::<T::Runtime>::free_balance(alice.clone()),
+        Balances::<T::Runtime>::free_balance(alice.clone())
```

Comment:
you ask for alice twice...but you are also not using `_bob_balance`... why are we asking for it?
```suggestion
        Balances::<T::Runtime>::free_balance(bob.clone())
```